tests/sflow/test_sflow.py

```diff
@@ -106,7 +113,14 @@ def get_ifindex(duthost, port):
 def config_sflow(duthost, sflow_status='enable'):
     duthost.shell('config sflow %s'%sflow_status)
     time.sleep(2)
+# ----------------------------------------------------------------------------------
 
+@pytest.fixture(scope='module')
+def config_sflow_feature(request, duthost):
+    # Enable sFlow feature on DUT if enable_sflow_feature argument was passed
+    if request.config.getoption("--enable_sflow_feature"):
+        duthost.shell("sudo config feature state sflow enabled")
+        time.sleep(2)
```

Comment:
Can you check that sflow is not enabled before enabling it? Also if the fixture enabled the sflow feature, can you disable it after the test is done?

Reply:
@yxieca I added check if sflow is not enabled, at teardown there is config_reload which will disable sflow feature even if I enabled it.